animata/widget/shopping-list.tsx

```
@@ -59,7 +59,11 @@ export default function ShoppingList({
   }[];
 }) {
   return (
-    <div className="h-64 w-48 rounded-3xl border bg-white p-4 font-sans shadow-sm">
+    <div
+      role="group"
+      tabIndex={0}
```

Biome (2.4.15)

[error] The HTML element div is non-interactive. Do not use tabIndex.

(lint/a11y/noNoninteractiveTabindex)